website/docs/docker.md

```diff
@@ -15,13 +15,13 @@ Docker builds are [automated](https://github.com/orhun/git-cliff/tree/main/.gith
 The easiest way of running **git-cliff** (in the git root directory with [configuration file](/docs/configuration) present) is to use the available tags from [Docker Hub](https://hub.docker.com/r/orhunp/git-cliff):
 
 ```bash
-docker run -t -v "$(pwd)/.git":/app/ "orhunp/git-cliff:${TAG:-latest}"
+docker run -t -v "$(pwd)":/app/ "orhunp/git-cliff:${TAG:-latest}"
```

Comment:
Isn't this breaking for users who were mounting `.git` before?

Reply:
I don't think so. The Git tooling mostly detects bare vs. full checkouts anyway. It was broken before because the Docker version when run this way didn't have any access to the config files. It only had access to the raw Git history and hence ignored any customization and use all defaults.